client/src/components/to-do-list/ToDoList.jsx

```diff
@@ -54,7 +54,7 @@ function ToDoList({isMobile, draggedEvent, setDraggedEvent, handleDragStart, tas
         <Button variant="contained" onClick={() => {openSetNewCat(true)}}>New Category</Button>
         <AddCategoryModal open={newCatModalOpen} closeCat={openSetNewCat} addCategory={addCategory} />
         <Button variant="contained" onClick={() => {openSetNewToDo(true)}}>New Task</Button>
-        <AddToDoModal open={newToDoModalOpen} closeCat={openSetNewToDo} addTodo={addTodo} info={taskData}/>
+        <AddToDoModal open={newToDoModalOpen} closeCat={openSetNewToDo} addTodo={addTodo} taskData={taskData}/>
```

Comment:
You don't have to change this, just some feedback on prop naming and readability. 

- I think having the word 'Set' in the name of a prop is confusing when using hooks. 
- Why is closeCat set to openSetNewToDo? Maybe just name it close ? 
- Why do you need a prop for open and close ? Couldn't you set open to false onClose ?

Reply:
I think it's an interesting approach, passing it that way. there may be slight performance gains.

the way i'm used to doing for readability is making a function that handles the setting of state, and then passing that function through props, instead of passing the setter itself.

Reply:
Again--good questions. Yes, could be cleaner, and should be.